tshirts.cpp

```diff
@@ -1,24 +1,31 @@
 #include <iostream>
 #include <assert.h>
 
-char size(int cms) {
+char getSizeName(int centimeter)
+{
     char sizeName = '\0';
-    if(cms < 38) {
+    if(centimeter <= 38)
+    {
         sizeName = 'S';
-    } else if(cms > 38 && cms < 42) {
+    }
+    else if(centimeter > 38 && centimeter <= 42)

```

Comment:
The condition `centimeter > 38` isn't necessary since it's already checked in the previous `if` and you have used an `else`.
Removing the condition ensures that changes would be made only in one place